### PR TITLE
Add GeoMesa Accumulo community module

### PR DESCRIPTION
**Pull request by elahrvivaz**

### CODE REVIEW

**Comment by eddumelendez** [comment]
Thanks for your contribution! This is available https://testcontainers.com/modules/geomesa-accumulo/

**Comment by elahrvivaz** [comment]
thank you!

File: modules/geomesa-accumulo/index.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+categories:
+  - nosql-database
+docs:
+  - id: java
+    url: https://github.com/geomesa/accumulo-uno
+    maintainer: community
+    example: |
+      ```java
+      var image = DockerImageName.parse("ghcr.io/geomesa/accumulo-uno:2.1.3");
+      var accumulo = new new AccumuloContainer(image).withGeoMesaDistributedRuntime();
```
Comment (mdelapenya) [inline modules/geomesa-accumulo/index.md:12]: I guess this is incorrect Java, isn't it?
```java
var accumulo = new new
```

Comment (elahrvivaz) [reply modules/geomesa-accumulo/index.md:12]: blerg, i'll put up an MR to fix it, thanks!

Comment (elahrvivaz) [reply modules/geomesa-accumulo/index.md:12]: https://github.com/testcontainers/community-module-registry/pull/104

Comment (eddumelendez) [reply modules/geomesa-accumulo/index.md:12]: fixed in https://github.com/testcontainers/community-module-registry/commit/0405416266cf423d59afc02308ae2856515f8946

**Pull request merged**